HouseRules_Essentials/Rules/CardAdditionOverriddenRule.cs

```diff
@@ -88,15 +89,23 @@ private static void Patch(Harmony harmony)
                 return;
             }
 
-            if (!_isChest)
+            if (_isPotionStand)
             {
+                // TODO: Add method to allow custom card loot for Potion Stands here
                 return;
             }
 
             var addCardToPieceEvent = (SerializableEventAddCardToPiece)request;
             var gameContext = Traverse.Create(__instance).Property<GameContext>("gameContext").Value;
 
             var pieceId = Traverse.Create(addCardToPieceEvent).Field<int>("pieceId").Value;
+            var cardSource = Traverse.Create(addCardToPieceEvent).Field<int>("cardSource").Value;
+
+            if (cardSource != (int)MotherTracker.Context.Energy && cardSource != (int)MotherTracker.Context.Chest)
+            {
```

Comment:
In case you prefer using enums, I believe you may be able to replace `...<int>`:
```cs
Traverse.Create(addCardToPieceEvent).Field<MotherTracker.Context>("cardSource")
```

which would give you back something of type `MotherTracker.Context` instead of an int.  Either way is good though.